validator/src/cli.rs

```diff
@@ -2775,6 +2775,12 @@ pub fn test_app<'a>(version: &'a str, default_args: &'a DefaultTestArgs) -> App<
                 .validator(is_parsable::<u64>)
                 .takes_value(true)
                 .help("Override the runtime's account lock limit per transaction"),
+        )
+        .arg(
+            Arg::with_name("enable_bigtable_ledger_upload")
+                .long("enable-bigtable-ledger-upload")
+                .takes_value(false)
+                .help("Enable ledger upload to BigTable"),
```

Comment:
Can you please move this up near `enable-rpc-bigtable-ledger-storage`?
Also, I think we prefer adding `.hidden(hidden_unless_forced())` to args like this to make it harder for someone to accidentally stomp their real bigtable data with TestValidator data.

Reply:
Done :)